savvy-transactions/schema.graphql

```diff
@@ -0,0 +1,49 @@
+type Deposit @entity(immutable: true) {
```

Comment:
Create a [Transaction](https://github.com/rkhadder/subgraphs/blob/ramsey-savvy-subgraph/subgraphs/savvy/setup/schemas/blockchainPrimitives.schemas.graphql#L25) type and an [Event](https://github.com/rkhadder/subgraphs/blob/ramsey-savvy-subgraph/subgraphs/savvy/setup/schemas/blockchainPrimitives.schemas.graphql#L59) interface. Each event should extend the event interface. [Here](https://github.com/rkhadder/subgraphs/blob/ramsey-savvy-subgraph/subgraphs/savvy/src/helpers/utils.ts#L43) is a helper for creating events.

This allows us to easily query for all transactions and events regardless of entity then we can also query for specific events.

Otherwise you lose precision in bundled transactions.